src/Paginator/KeysetPaginator.php

```diff
@@ -296,4 +297,19 @@ private function previousPageExist(ReadableDataInterface $dataReader, Sort $sort
         }
         return false;
     }
+
+    private function getValueFromItem($item, string $field)
+    {
+        $getterName = $this->getGetterName($field);
+        if (is_object($item) && method_exists($item, $getterName)) {
```

Comment:
```suggestion
        if (is_object($item) && is_callable([$item, $getterName])) {
```
https://3v4l.org/JGgoM

Reply:
fixed